nvflare/private/fed/app/simulator/simulator_runner.py

```diff
@@ -348,7 +349,12 @@ def run(self):
 
     def run_processs(self, return_dict):
         # run_status = self.simulator_run_main()
-        run_status = mpm.run(main_func=self.simulator_run_main, shutdown_grace_time=3, cleanup_grace_time=6)
+        try:
+            run_status = mpm.run(
+                main_func=self.simulator_run_main, run_dir=self.workspace, shutdown_grace_time=3, cleanup_grace_time=6
+            )
+        except Exception:
+            run_status = ProcessExitCode.EXCEPTION
```

Comment:
Could we also print/log the exception message here?
Some users have run into this scenario but we don't 100% know what happens, it might be helpful if we can see the exception and traceback.

Reply:
Added log.